scalecube-organization/src/main/java/io/scalecube/organization/OrganizationServiceImpl.java

```diff
@@ -89,12 +93,15 @@ public static Builder builder() {
                 .secretKey(secretKey)
                 .build());
         KeyStoreFactory.get().store(organization.secretKeyId(), secretKey);
-        result.success(new CreateOrganizationResponse(organization.id(),
+        CreateOrganizationResponse response = new CreateOrganizationResponse(organization.id(),
             organization.name(),
             organization.apiKeys(),
             organization.email(),
-            organization.ownerId()));
+            organization.ownerId());
+        logger.debug("createOrganization: exit, return: {}", response);
+        result.success(response);
       } catch (Throwable ex) {
+        logger.error("createOrganization: error: {}, request: {}", ex, request);

```

Comment:
@segabriel 
try to use "insert a suggestion" (new github feature) to propose some code changes which will be automatically applied to code if author of pull request accepts this suggestion

For example:
```suggestion
        logger.error("createOrganization with request: {}, error: {} ", request, ex);
```